TERMINOLOGY.md

```diff
@@ -1,53 +1,70 @@
-# Terminology
+# Concepts and terminology
 
 ## Estimators and predictors
+
 Fairlearn largely follows the [terminology established by scikit-learn](https://scikit-learn.org/stable/developers/contributing.html#different-objects), specifically:
-- Estimator: implements a `fit` method
-- Predictor: implements a `predict` method.
-Unlike [scikit-learn estimators](https://scikit-learn.org/stable/glossary.html#term-estimator) our estimators in fairlearn may not be deterministic. Lots of disparity mitigation techniques require randomized output to meet their objective in expectation. We do provide unrandomized versions of every method as well, but they may not be able to uphold the same theoretical guarantees.
+- _Estimators_ implement a `fit` method.
+- _Predictors_ implement a `predict` method.
+
+
+**Randomization.** Unlike [scikit-learn estimators](https://scikit-learn.org/stable/glossary.html#term-estimator), estimators in fairlearn can produce randomized predictors. Randomization of predictions is required to satisfy many definitions of fairness. Because of randomization, it is possible to get different outputs from the predictor's `predict` method on identical data. For each of our methods, we provide explicit access to the probability distribution used for randomization.
+
+## Fairness of AI systems
+
+AI systems can behave unfairly for a variety of reasons. Sometimes it is because of societal biases reflected in the training data and in the decisions made during the development and deployment of these systems. In other cases, AI systems behave unfairly not because of societal biases, but because of characteristics of the data (e.g., too few data points about some group of people) or characteristics of the systems themselves. Because it can be hard to distinguish between these reasons (and these reasons are not mutually exclusive and often exacerbate one another), we define whether an AI system is behaving unfairly in terms of its impact on people – i.e., in terms of harms – and not in terms of specific causes, such as societal biases, or in terms of intent, such as prejudice.
```

Comment:
didn't we want to stay away from the term bias?

Reply:
oh, below you're talking about that. Did you use it here because this is not related to fairlearn yet?

Reply:
Yes... I wanted to get across why we are not talking about debiasing, but also acknowledge that societal biases exist.

Reply:
@romanlutz : does this need to be moved or edited?

Reply:
keeping as is.